redhat/lyrionmusicserver.spec

```diff
@@ -539,7 +491,7 @@ else
 	setSystemd
 fi
 
-PORT=`awk '/^httpport/ {print $2}' %{_var}/lib/%{shortname}/prefs/server.prefs`
+PORT=`/usr/bin/perl -lane  'if ( /^httpport:/) {print $F[1]}' %{_var}/lib/%{shortname}/prefs/server.prefs`
```

Comment:
We can save a bit of processing here by exiting after the first match.

```suggestion
PORT=`/usr/bin/perl -lane  'if ( /^httpport:/) {print $F[1]; exit}' %{_var}/lib/%{shortname}/prefs/server.prefs`
```

Reply:
Yes, I'll fix this too.